README.md

```diff
@@ -39,7 +44,7 @@ Remember to instruct PHPUnit to load extensions in your `phpunit.xml`:
 
 ## Usage
 
-Enable the globals annotation extension in your PHPUnit configuration:
+Enable the globals attribute extension in your PHPUnit configuration:
```

Comment:
I have changed the docs to be based on attributes but with fallback on annotation